README.md

```diff
@@ -1072,6 +1072,306 @@ We recommend that you can write your own `CMakeUserPresets.json` file. You can i
 }
 ```
 
+
+## Cheat sheet: autotools vs. cmake
+
+Assume that `$oomph_home_dir` is the `oomph-lib` home directory:
+<table>
+<tr><th>autotools</th><th>cmake</th></tr>
+
+<tr>
+<td colspan="2">
+<b>- Build and install the entire library using a helper script:</b>
+</td>
+</tr>
+<tr>
+<td>
+<pre>
+# Go to home directory
+cd $oomph_home_dir<br>
+# Run the autogen script which prompts
+# for configure options
+./autogen.sh
+
+</pre>
+</td>
+<td>
+<pre>
+# Go to home directory
+cd $oomph_home_dir<br>
+# Run the hierher script which prompts
+# for configure options
+./something_similar.sh
+</pre>
+</td>
+</tr>
+
+
+<tr>
+<td colspan="2">
+<b>- Alternative: Build and install the entire library step by step:</b>
+</td>
+</tr>
+<tr>
+<td>
+<pre>
+# Go to home directory
+cd $oomph_home_dir<br>
+# Run manual config/build/install 
+# sequence
+./configure hierher check this
```

Comment:
Nope; it's on my todo  list. Once it's all settled and we've agreed on the overall workflow I'll warp this all up in a script. You'd be the wrong person to write this because you know (too well!) how it all works without handholding! You want a newbie to write this (and have somebody who understands it to check!).

Reply:
Understood!